homeassistant/components/knx/light.py

```diff
@@ -269,8 +269,16 @@ def supported_features(self):
         update_white_value = ATTR_WHITE_VALUE in kwargs
         update_color_temp = ATTR_COLOR_TEMP in kwargs
 
-        # always only go one path for turning on (avoid conflicting changes
```

Comment:
So now you will go down multiple paths and have conflicting changes?

Reply:
I think the source of the conflicting changes was sending an ON telegram after eg. brightness so the actuators default turn-on-value overwrote the brightness telegram.

We only send an ON telegram if no other light option is updated. (Line 273). 

Only problem I could imagine is actuator specific. Some don't turn on when receiving a color-temperature telegram only. They expect either ON or brightness. But I think HA ui doesn't allow to set color-temperature on a turned off light.